media-video/guvcview/files/guvcview-2.2.1-fix-desktop-file-for-cmake.patch

```diff
@@ -0,0 +1,85 @@
+From 39f76eda4d66ef7b2303ce795e701e43fafadff2 Mon Sep 17 00:00:00 2001
```

Comment:
Always include links to where the patch came from at the top too (we can guess based on the hash but let's make it easy).

Reply:
From upstream, yes.
Patch commented.